mkdocs_monorepo_plugin/merger.py

```diff
@@ -55,8 +55,8 @@ def merge(self):
                 source_dir = docs_dir
                 dest_dir = self.temp_docs_dir.name
             else:
-                source_dir = "{}/docs".format(docs_dir)
-                dest_dir = "{}/{}".format(self.temp_docs_dir.name, alias)
+                source_dir = os.path.join(docs_dir, "docs")
+                dest_dir = os.path.join(self.temp_docs_dir.name, alias.replace("/", "-"))
 
```

Comment:
@bih for some reason this change trigger a `A relative path to ... is included in the 'nav' configuration, which is not found in the documentation files`. Not sure what could have triggered that but it only appears in 0.4.9 and works fine in 0.4.8

Reply:
Ah I figured it out. It occurs if the site name has a `/` in it. If that case, it converts the `/` into `-`, which breaks the path

Reply:
@emmaindal

Reply:
@rossmechanic Ah interesting, thanks for letting us know. We can investigate. Could you create a GitHub issue for this?

By the way, if you want to receive more direct support from us, I would recommend joining the [Backstage discord](https://discord.gg/MUpMjP2) and hang out in the #docs-like-code channel.